web/src/components/organisms/Settings/Account/index.tsx

```diff
@@ -0,0 +1,53 @@
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Input from "@reearth-cms/components/atoms/Input";
+import { useCallback, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import useHooks from "./hooks";
+
+const Account: React.FC = () => {
+  const { me, updateName } = useHooks();
+  const navigate = useNavigate();
+
+  const [filterText, setFilterText] = useState("");
+
+  const handleInputChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      setFilterText?.(e.currentTarget.value);
+    },
+    [setFilterText]
+  );
+
+  const handleClick = useCallback(() => {
+    updateName(filterText);
+  }, [filterText, updateName]);
```

Comment:
Since you already have a hooks.ts file, can you go the whole nine yards and move all this into there as well?